File: server/php/src/Infrastructure/Mappers/PaymentDetails/OperationOutputMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\OperationOutput as OperationOutputDto;
use OnlinePayments\Sdk\Domain\OperationOutput as OperationOutputSdk;

final class OperationOutputMapper
{
    public static function mapFromSdkResponse(?OperationOutputSdk $response): OperationOutputDto
    {
        $dto = new OperationOutputDto();
        $dto->id = $response?->id ?? null;
        $dto->operationReferences = OperationPaymentReferencesMapper::mapFromSdkResponse($response?->operationReferences ?? null);
        $dto->paymentMethod = $response?->paymentMethod ?? null;
        $dto->statusOutput = PaymentStatusOutputMapper::mapFromSdkResponse($response?->statusOutput ?? null);
        $dto->amountOfMoney = AmountOfMoneyMapper::mapFromSdkResponse($response?->amountOfMoney ?? null);
        $dto->references = PaymentReferencesMapper::mapFromSdkResponse($response?->references ?? null);
        $dto->status = $response?->status ?? null;

        return $dto;
    }
}

File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/OperationOutputMapper.cs
using OperationOutputDto = Business.Domain.Payments.PaymentDetails.OperationOutput;
using OperationOutputSdk = OnlinePayments.Sdk.Domain.OperationOutput;

namespace Infrastructure.Mappers.PaymentDetails;

public static class OperationOutputMapper
{
    public static OperationOutputDto Map(OperationOutputSdk? response)
    {
        return new()
        {
            Id = response?.Id,
            OperationReferences = OperationPaymentReferencesMapper.Map(response?.OperationReferences),
            PaymentMethod = response?.PaymentMethod,
            StatusOutput = PaymentStatusOutputMapper.Map(response?.StatusOutput),
            AmountOfMoney = AmountOfMoneyMapper.Map(response?.AmountOfMoney),
            References = PaymentReferencesMapper.Map(response?.References),
            Status = response?.Status,
        };
    }
}